proposals/lip-0018.md

```diff
@@ -190,7 +190,7 @@ The type of the reclaim transaction is the lowest positive integer that is not u
 
 ###### Schema
 
-The `asset` property of a reclaim transaction needs to contain the `amount` property which is of type _string_. The value of `asset.amount` must represent a decimal number and this number must be equal to the balance associated to the address of the current format in Beddows.
+The `asset` property of a reclaim transaction needs to contain the `amount` property which is of type _uint64_. The value of `asset.amount` must be equal to the balance associated to the address of the current format in Beddows.
```

Comment:
This LIP was written when we did not have protobuf serialization and Lisk amounts in JSON were strings. I don't think it adds much clarification to change it here and rather makes the LIP inconsistent. Later in the serialization section we write " interpreted as a 64-bit signed integer" for example. LIP 0028 already defines all the serializations, also for the reclaim transaction.

Reply:
Changed the serialization to say "unsigned integer". This way, the types are consistent with future LIPs, which I think is good if anyone checks this LIP to have info on the reclaim transaction.

Reply:
Changed the serialization section to say "unsigned integer". I understand your point. I think that here it is worth changing the type of `amount` to be more coherent with future LIPs.

Reply:
That sounds also good.